docs/Guides/Scim/Guide-to-sync-users-in-vendasta.md

```diff
@@ -0,0 +1,42 @@
+# Guide to use Vendasta SCIM APIs to sync Users
+
+
+## Overview
+System for Cross-domain Identity Management ([SCIM](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management)) focuses on syncing user accounts and permissions between systems but has an extension system that allows syncing any type of record. There are several well known endpoint paths for discovering what the server supports and resource schema definitions.
```

Comment:
Shall we add what this guides covers in the overview section? It now just describes the SCIM.  It would be helpful to the readers if we provide an outline of what is covered inside this guide.